src/common/xo/index.js

```diff
@@ -634,6 +634,60 @@ export const editNetwork = ({ id }, props) => (
   call('network.set', { ...props, id })
 )
 
+import CreateNetworkModalBody from './create-network-modal'
+export const createNetwork = container => (
```

Comment:
It is either the pool or the host on which you want to create the network.
If it's a pool: the network will be created on this pool and the modal will propose PIFs from the `pool.master` to be attached to the network.
If it's a host: the network will be created on its pool and the modal will propose PIFs from this host.